ChopChop/ViewModels/OnlineRecipes/OnlineRecipeBySelfViewModel.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+import Combine
+
+class OnlineRecipeBySelfViewModel: OnlineRecipeViewModel {
+
+    func onDelete() {
+        do {
+            try storageManager.removeRecipeFromOnline(recipe: recipe)
+        } catch {
+            assertionFailure("Could not remove recipe from online")
```

Comment:
What happens in the front end in this case?

Reply:
Technically it should never fail, thus put an assertionFailure